WP_Auth0.php

```diff
@@ -31,43 +31,12 @@
 
 define( 'WPA0_LANG', 'wp-auth0' ); // deprecated; do not use for translations
 
-require_once WPA0_PLUGIN_DIR . 'vendor/autoload.php';
-require_once WPA0_PLUGIN_DIR . 'functions.php';
+require_once __DIR__ . '/vendor/autoload.php';
 
 /*
  * Startup
  */
 
-function wp_auth0_autoloader( $class ) {
```

Comment:
Deleting this function is technically a breaking change. Let's `return false` and mark as deprecated.

Reply:
Done.